server/routes/title.js

```diff
@@ -80,5 +106,22 @@ router.route('/seed').post((req, res) => {
 });
 
 
+router.route('/scaled-seed/:currentCounts').post(async (req, res) => {
+
+  let totalTittles = 11000000;
+  if (req.params.counts > totalTittles) { return res.status(200).json(`Exceeded max number of tittles(${totalTittles})`); }
```

Comment:
For production, this should probably return a `400 Bad Request` or a `404 Not Found` instead of a 200 since it's exceeding the range of acceptable titles.

Reply:
Yes, thank you. Good catch!